### PR TITLE
reSTified a README that slipped through the cracks

### PR DESCRIPTION
**Pull request by ben-albrecht**
module_search -> module_search.rst (Lydia can content-review)

### CODE REVIEW

**Comment by ben-albrecht** [comment]
Summoning Lydia @lydia-duncan

File: doc/release/technotes/README.module_search.rst [hunk 1]
```diff
@@ -6,30 +7,34 @@ This README describes how modules are located by the current Cray implementation
...
 Internal modules are always loaded by a chapel program *as if* the
-statement "use ChapelStandard;" has been inserted ahead of the first
+statement ``use ChapelStandard;`` has been inserted ahead of the first
 statement in the main module.  Unlike standard modules, internal modules cannot
-be overridden.  
+be overridden.
 
 The compiler uses three module search paths: one for internal modules, one for
 standard modules and one for user modules.  The user path first includes all the
 paths used to specify module files directly on the "chpl" command line and
```
Comment (lydia-duncan) [inline doc/release/technotes/README.module_search.rst]: Should we update the "chpl" reference as well?

**Comment by lydia-duncan** [comment]
:+1:

File: doc/release/technotes/README.module_search.rst (hunk 1 above)
Comment (ben-albrecht) [inline doc/release/technotes/README.module_search.rst]: Yes. Done.

**Pull request merged**